src/lib.rs

```diff
@@ -884,6 +884,9 @@ impl Url {
 
     /// Return this URL’s fragment identifier, if any.
     ///
+    /// A fragment is the part of a url following the `#` symbol,
```

Comment:
URL should be capitalized as an initialism because it appears that way in all the rest of the documentation in this crate.